deps/imgui-glew/imgui_impl_glfw_gl3.cpp

```diff
@@ -395,7 +395,7 @@ void ImGui_ImplGlfwGL3_NewFrame()
     g_MouseWheel = 0.0f;
 
     // Hide OS mouse cursor if ImGui is drawing it
-    glfwSetInputMode(g_Window, GLFW_CURSOR, io.MouseDrawCursor ? GLFW_CURSOR_HIDDEN : GLFW_CURSOR_NORMAL);
+    //glfwSetInputMode(g_Window, GLFW_CURSOR, io.MouseDrawCursor ? GLFW_CURSOR_HIDDEN : GLFW_CURSOR_NORMAL);
```

Comment:
Eh i figure commenting it will make it clearer that it was there. I added imgui directly in because i had to change a bunch of stuff and its small. i thought about making my own repo and including it, but it would have taken more work lol